python/pyraphtory/poetry.lock

```diff
@@ -0,0 +1,401 @@
+[[package]]
```

Comment:
This can be removed can't it?

Reply:
Poetry docs suggest to commit it so everyone ends up with the same packages, not sure it is that helpful.